Application/DTO/BaseEntityDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.DTO
+{
+    public class BaseEntityDto
```

Comment:
what the purpose of have it?

Reply:
just practicing inheritance and also for sure another models also will have Id, so will use this everywhere